docs/reference/compatibility/data-streams.md

```diff
@@ -0,0 +1,87 @@
+---
+navigation_title: Data streams comparison
+description: Learn how EDOT optimizes telemetry storage and query performance in Elastic Observability compared to classic APM.  
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+  - id: edot-sdk
+---
+
+# EDOT data streams compared to classic APM
+
+The Elastic Distribution of OpenTelemetry (EDOT) stores telemetry data using a storage model optimized for OpenTelemetry signals. When `mapping_mode: otel` is enabled, EDOT writes logs, traces, and metrics to specialized data streams aligned with OpenTelemetry semantics to improve efficiency.
+
+This architecture is designed for scalable observability workloads. It supports dynamic attributes, reduces mapping complexity, and avoids issues like mapping explosions or manual dimension setup.
+
+EDOT uses Elasticsearch’s [Logs data stream (LogsDB)](docs-content://manage-data/data-store/data-streams/logs-data-stream.md) and [Time Series Data Streams (TSDS)](docs-content://manage-data/data-store/data-streams/time-series-data-stream-tsds.md) as storage backends. These are purpose-built to handle the scale and variety of observability data.
+
+## Storage architecture
+
+EDOT uses specialized index modes for each type of telemetry signal:
+
+| Signal type | Index mode    | Storage backend |
+|-------------|---------------|------------------|
+| Logs        | `logsdb`      | LogsDB           |
+| Traces      | `logsdb`      | LogsDB           |
+| Metrics     | `time_series` | TSDS             |
+
```

Comment:
Maybe remove this section? It adds little beyond what's stated in the previous sentence.